lib/ansible/plugins/doc_fragments/hwc.py

```diff
@@ -0,0 +1,78 @@
+# Copyright: (c) 2018, Huawei Inc.
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment(object):
+
+    # HWC doc fragment.
+    DOCUMENTATION = '''
+options:
+    identity_endpoint:
+        description:
+            - The Identity authentication URL.
+        type: str
+        required: true
+    user_name:
+        description:
+            - The user name to login with.
+        type: str
+        required: true
+    password:
+        description:
+            - The password to login with.
+        type: str
+        required: true
+    domain_name:
+        description:
+            - The name of the Domain to scope to (Identity v3).
+        type: str
+        required: true
+    project_name:
+        description:
+            - The name of the Tenant (Identity v2) or Project (Identity v3).
+        type: str
+        required: true
+    region:
+        description:
+            - The region to which the project belongs.
+        type: str
+        required: true
+    timeouts:
+        description:
+            - The timeouts for create/update/delete operation.
+        type: dict
+        suboptions:
+            create:
+                description:
+                    - The timeouts for create operation.
+                type: str
+                default: '10m'
+            update:
+                description:
+                    - The timeouts for update operation.
+                type: str
+                default: '10m'
+            delete:
+                description:
+                    - The timeouts for delete operation.
+                type: str
+    id:
+        description:
+            - The id of resource to be managed.
+        type: str
+notes:
+  - For authentication, you can set identity_endpoint using the
+    C(IDENTITY_ENDPOINT) env variable.
```

Comment:
Same changes above would be required here, too.

Reply:
Done